### PR TITLE
Fsi updates 011521c

### PR DESCRIPTION
**Pull request by jbranson000**
#1017 
Included the following Financial Services mixins:
Social Network Usage Details, Support Request, Support Site Search, Survey Response Details, Trade Transaction Details, User Account  Details, User Login Process, & Watchlist Steps.

### CODE REVIEW

File: components/mixins/experience-event/experienceevent-support-request.schema.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  "meta:abstract": true,
+  "meta:tags": {
+    "industry": ["financial_services", "telecom", "high_tech", "public_sector"]
+  },
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"],
+  "description": "",
+  "definitions": {
+    "support-request": {
+      "properties": {
+        "xdm:": {
```
Comment (mjasrotia) [inline components/mixins/experience-event/experienceevent-support-request.schema.json:22]: Object name missing

File: components/mixins/experience-event/experienceevent-survey-response-details.schema.json [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  "description": "",
+  "definitions": {
+    "survey-response-details": {
+      "properties": {
+        "xdm:customerFeedback": {
+          "type": "object",
+          "properties": {
+            "xdm:surveyAnswers": {
+              "title": "Survey Answers",
+              "type": "array",
```
Comment (mjasrotia) [inline components/mixins/experience-event/experienceevent-survey-response-details.schema.json:39]: Not the right way to define arrays

File: components/mixins/experience-event/experienceevent-survey-response-details.schema.json [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+              "description": "An array of the answers provided by the visitor within a survey."
+            },
+            "xdm:surveyID": {
+              "title": "Survey ID",
+              "type": "string",
+              "description": "ID for the specific survey taken."
+            },
+            "xdm:surveyQuestions": {
+              "title": "Survey Questions",
+              "type": "array",
```
Comment (mjasrotia) [inline components/mixins/experience-event/experienceevent-survey-response-details.schema.json:49]: not the right way to define arrays

File: components/mixins/experience-event/experienceevent-user-account-details.schema.json [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+              "description": "Captures the user account type, if applicable."
+            },
+            "xdm:userAccountRegistration": {
+              "title": "User Account Registration",
+              "$ref": "https://ns.adobe.com/xdm/datatypes/tool-usage",
+              "description": "Extension of toolUsage data type."
+            },
+            "xdm:userAccountPreferences": {
+              "title": "User Account Preferences",
+              "type": "array",
```
Comment (mjasrotia) [inline components/mixins/experience-event/experienceevent-user-account-details.schema.json:76]: not the right way to define arrays

**Review by mjasrotia — APPROVED** [review]

**Review by prabhum2 — APPROVED** [review]

**Pull request merged**